app/javascript/styles/mastodon/components.scss

```
@@ -299,6 +299,10 @@
     }
   }
 
+  &--with-counter {
+    padding-right: 4px;
```

This should be changed to account for RTL layouts:
```suggestion
    padding-inline-end: 4px;
```